content/applications/sales/sales/send_quotations/terms_and_conditions.rst

```diff
@@ -1,117 +1,153 @@
 
-.. image:: terms_and_conditions/terms_conditions_4.png
-   :align: center
-   :class: img-thumbnail
-   :alt: General Terms & Conditions on your website
+On the desired quotation template form, click :guilabel:`Edit`. Under the :guilabel:`Confirmation`
+tab, there are options to activate an :guilabel:`Online Signature`, :guilabel:`Online Payment`, and
+:guilabel:`Confirmation Mail`.
+
+Click on the :guilabel:`Confirmation Mail` field to reveal a drop-down menu from which an email
+template can be chosen or created. To create a new email template from here, start typing the name
+of this new template, and then select :guilabel:`Create and Edit...`, which reveals a
+:guilabel:`Create: Confirmation Mail` pop-up form.
 
-You can refer to this page in the footer of all your documents. The layout is available in the
-**General Settings** under the **Business Documents** category. For example, this footer appears
-in every document from Odoo.
+On this pop-up form, customize the email message in a number of different ways, but be sure to
+click :guilabel:`Attachments` at the bottom of the :guilabel:`Content` tab to add the desired
+terms and conditions attachment to the mailing. Once done, click :guilabel:`Save` to save all
+changes.
 
-.. image:: terms_and_conditions/terms_conditions_7.png
+.. image:: terms_and_conditions/terms-conditions-create-edit.png
    :align: center
-   :class: img-thumbnail
-   :alt: General Terms & Conditions in your business documents
+   :alt: Terms and conditions as attachment in quotation templates.
+
+Terms and conditions on a website
+=================================
+
+Use the Odoo *Website* application to create a terms and conditions page. To do that, first navigate
+to the front-end of the website, via :menuselection:`Website app --> Go to Website`. From here,
+click :menuselection:`+ New` in the top right corner of the page, then select :menuselection:`Page`.
+Give the terms and conditions page a title in the pop-up form that appears.
 
-GT&C as attachment in your emails
----------------------------------
+Then, decide if this page should be accessible via the main Website header menu, using the
+:guilabel:`Add to menu` toggle in the pop-up form, and click :guilabel:`Continue` to create the new
+page.
 
-Attach an external document with your general terms and conditions when you are about to send your
-quotation by email to your customers.
+Doing so reveals a blank page on the website that can be fully customized and edited with Odoo's
+drag-and-drop building blocks. Use those building blocks to add the terms and conditions to the
+page, and click :guilabel:`Save` to save all changes.
 
-.. image:: terms_and_conditions/terms_conditions_5.png
+Don't forget to publish the page, via the :guilabel:`Unpublished` > :guilabel:`Published` toggle at
+the top of the page. Unpublished pages will **not** be accessible to website visitors.
+
+.. image:: terms_and_conditions/terms-conditions-page-frontend.png
```

Comment:
This image _could_ be retaken to be a bit more legible. I'd suggest reducing the window width and zooming in (`CTRL +`)